system-int-test/test/cmr/system_int_test/search/collection_search_format_test.clj

```diff
@@ -695,6 +687,42 @@
                                            {:url-extension "json"})
                 [:status :results])))))))
 
+(deftest search-collection-csv
+  (let [c1 (d/ingest "PROV1" (dc/collection {:short-name "shortie number 1"
+                                              :version-id "V1"
+                                              :long-name "lng nme"
+                                              :processing-level-id "L1"
+                                              :entry-title "I am #1"
+                                              :beginning-date-time "1970-01-01T12:00:00Z"
+                                              :platforms [{:short-name "platform #1"
+                                                           :long-name "platform #1"
+                                                           :type "t"}]}))
+        c2 (d/ingest "PROV2" (dc/collection {:short-name "world's shortest name: so short you won't believe your eyes!"
+                                              :version-id "V2"
+                                              :long-name "world's longest name"
+                                              :processing-level-id "L2"
+                                              :entry-title "lorem ipsum"
+                                              :beginning-date-time "2000-01-01T12:00:00Z"
+                                              :platforms [{:short-name "platform #21"
+                                                           :long-name "platform #21"
+                                                           :type "good platform"}
+                                                          {:short-name "platform #32"
+                                                           :long-name "platform #32"
+                                                           :type "very good platform"}]}))
+        g1 (d/ingest "PROV1" (dg/granule c1))])
+  (index/wait-until-indexed)
+
+  (let [response (search/find-concepts-csv :collection {:include-granule-counts "true"})]
```

Comment:
We have `granule-count` always as a field in CSV response. Its value should always be included in the result rather than via a different flag.